odoo/addons/base/tests/test_create.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo.tests import tagged
+
+from odoo.addons.base.tests.common import PerformanceCommon
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install', '-at_install', 'prout')
+class TestCreate(PerformanceCommon):
+
+    def test_create_performance(self):
+        """Enforce create methods always support batch creation for standard models"""
+        errors = []
+        for model in self.env:
```

Comment:
Do we really need to also enforce this constraint for transient models ? :thinking:

Reply:
Probably not.